scripts/collection_from_items.py

```diff
@@ -22,6 +22,12 @@ class NoItemsError(Exception):
     pass
 
 
+def linz_slug_type(value: str) -> str:
+    if not value:
```

Comment:
Do we also want to validate that it contains only [valid slug characters](https://github.com/linz/argo-tasks/blob/68940d582004f88350b30e8a6064dd1174e2631e/src/utils/slugify.ts#L15-L15)?